client/src/components/friends-page/AddButton.css

```diff
@@ -35,6 +37,12 @@
   height: auto;
 }
 
+.stats-input {
+  border-radius: 5px;
+  width: 100%;
+  height: auto;
```

Comment:
The initial value _is_ [auto](https://developer.mozilla.org/en-US/docs/Web/CSS/height)